allennlp/models/basic_classifier.py

```diff
@@ -139,7 +141,8 @@ def decode(self, output_dict: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor
         classes = []
         for prediction in predictions_list:
             label_idx = prediction.argmax(dim=-1).item()
-            label_str = self.vocab.get_token_from_index(label_idx, namespace="labels")
+            label_str = \
```

Comment:
our style is not to use `\` for continuation, instead you could do it fluent-style in parentheses

```
label_str = (self.vocab.get_index_to_token_vocabulary(self._label_namespace)
             .get(label_idx, str(label_idx)))
```